src/proc.h

```diff
@@ -33,6 +33,11 @@ int print_proc_maps(int pid);
 int get_threads(int pid, int **tids);
 
 /*
+ * get thread states
```

Comment:
Useful comment, lol.

Reply:
Made it more verbose: https://github.com/tbricks/tbstack/pull/14/commits/aa71da66d6faad162b6f9e6307e10b79bb0adcf4. Some comments in this header are made just for consistency.